src/components/QuestionEdit/index.tsx

```diff
@@ -41,6 +41,16 @@ export default function QuestionEdit({
   const { showAlert } = useStore();
   const options = watch('options');
   const isWithoutOptions = options.length === 1;
+  const { points, text } = watch();
+
+  useEffect(() => {
+    onQuestionChange({
+      ...watch(),
```

Comment:
Is it safe to call watch here inside an useEffect?

Reply:
Or we can use the value from watch from the above? We should probably call watch once to avoid perf issues.

Reply:
using `...rest` to get rest data